webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -132,6 +275,14 @@ export class WebhookDisplayTable extends Component {
                 title='Webhook created successfully.'>
               </InlineNotification>
             )}
+
+          {this.state.deleteEvent && (
```

Comment:
Could you not reuse the showNotificationOnTable property and then set the options as needed either the defaults for the webhook creation or your vars if deletion?  Might be able to reduce this code a bit.